src/components/ConcurrencyLimitMenu.vue

```diff
@@ -2,13 +2,15 @@
   <p-icon-button-menu v-bind="$attrs">
     <copy-overflow-menu-item label="Copy ID" :item="concurrencyLimit.id" />
     <p-overflow-menu-item v-if="can.delete.concurrency_limit" label="Delete" @click="open" />
+    <p-overflow-menu-item v-if="can.delete.concurrency_limit" label="Reset" @click="openReset" />
```

Comment:
Curious why the `delete` and not `update` permission check here. Assuming that's just following what the api allows?

Reply:
Ah.  Good call!